package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@sysvale/component-library",
-  "version": "1.0.14",
+  "version": "1.15.0",
```

As this PR adds a new functionality we can just increment the MINOR version.

```suggestion
  "version": "1.1.0",
```